src/main.cpp

```diff
@@ -162,15 +162,15 @@ int main(int argc, char *argv[]) {
     std::unique_ptr<EducationOptions> eduOptions (new EducationOptions(resourcePackManager));
     ServerInstanceEventCoordinator instanceEventCoordinator;
     std::unique_ptr<ServerInstance> instance (new ServerInstance(minecraftApp, instanceEventCoordinator));
-    LauncherV8Platform::initVtable(handle);
-    LauncherV8Platform v8Platform;
-    v8::V8::InitializePlatform((v8::Platform*) &v8Platform);
-    v8::V8::Initialize();
-    instance->initializeServer(minecraftApp, whitelist, &permissionsFile, &pathmgr, idleTimeout, props.worldDir.get(), props.worldName.get(), props.motd.get(), levelSettings, props.viewDistance, true, { props.port, props.portV6, props.maxPlayers }, props.onlineMode, {}, "normal", *mce::UUID::EMPTY, eventing, resourcePackRepo, ctm, *resourcePackManager, createLevelStorageFunc, pathmgr.getWorldsPath(), nullptr, mcpe::string(), mcpe::string(), std::move(eduOptions), resourcePackManager, [](mcpe::string const& s) {
```

Comment:
pathmgr.getWorldsPath() is broken on armhf, my hotifx replaces it with appPlatform->getCurrentStoragePath() + "/worlds/", but its not the same path

Reply:
Please use getWorldsPath(), at least on i386 (you can ifdef it for arm, I'll accept the PR then).

Reply:
Actually, I'll do it myself